test/libsolidity/syntaxTests/inheritance/modfiers_in_constructor_contetxt.sol

```diff
@@ -0,0 +1,9 @@
+// This generated an invalid warning on m1 in some compiler versions.
```

Comment:
There's a typo in the filename. Also indentation seems to be inconsistent with our existing tests.